.github/workflows/release.yml

```diff
@@ -15,11 +15,16 @@ jobs:
           pip install -r requirements.txt
           sudo apt-get install -y gettext
 
-      - name: tx push / pull
+      - name: tx sync (push -> pull)
         if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
+        env:
+          TX_TOKEN: ${{ secrets.TX_TOKEN }}
         run: |
-          TX_TOKEN=${{ secrets.TX_TOKEN }} make transifex_sync # push
-          TX_TOKEN=${{ secrets.TX_TOKEN }} make transifex_pull
+          curl -OL https://github.com/transifex/cli/releases/download/v1.3.1/tx-linux-amd64.tar.gz
+          tar -xvzf tx-linux-amd64.tar.gz
+          ./tx push
+          ./tx pull
+          ./tx status
```

Comment:
@why-not-try-calmer is it possible, that the Makefile did more than just push and pull (like extracting translatable strings from docs?)
Maybe it's safer to call `tx` inside the Makefile and preserve those dependencies.

Reply:
On it.